<!DOCTYPE html>

<html class="gr__princeton_edu">

<head>
	<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
		
		<title>Haoyang Zeng - Home</title><meta name="description" content="Website for Haoyang Zeng">
	
	<meta name="keywords" content="Computational biology, machine learning, deep learning">

	<meta name="viewport" content="width=device-width, initial-scale=1.0">

	<link rel="stylesheet" type="text/css" href="./data/main_style.css">
	<link href="./data/css" rel="stylesheet">

	<!-- Global site tag (gtag.js) - Google Analytics -->
	<script type="text/javascript" async="" src="./data/analytics.js"></script><script async="" src="./data/js"></script>
	<script>
	  window.dataLayer = window.dataLayer || [];
	  function gtag(){dataLayer.push(arguments);}
	  gtag('js', new Date());

	  gtag('config', 'UA-68992915-1');
	</script>
</head>


<!--- Header -->
<div id="header">
	<label for="nav-trigger" class="nav-trigger hamburger">
		<span class="open-btn">
			<span class="mobile"></span>
			<span class="mobile"></span>
			<span class="mobile"></span>
		</span>
	</label>
	<div id="sitename">
		<span class="wsite-logo">

		<a href="http://haoyangz.github.io">

		    <span id="wsite-title" style="position:relative; top:5px">Haoyang Zeng</span><img src="./data/profile.png" style="border-radius: 50%; width: 6%;margin-left: 2%; vertical-align: middle">

		</a>

		</span>
	</div>
</div>

<div id="wrapper">
	<div class="bg-wrapper">
		<div id="navigation"><ul class="wsite-menu-default">
	<li id="active" class="wsite-menu-item-wrap">
		<a href="http://haoyangz.github.io/#home" class="wsite-menu-item">
			Home
		</a>
		
	</li>
	<li id="pg191988064268116182" class="wsite-menu-item-wrap">
		<a href="http://haoyangz.github.io/#pub" class="wsite-menu-item">
			Publication
		</a>
		
	</li>
</ul>
</div>
			

<!--- Bio -->

<div id="content-wrapper">
	<div id="content"><div id="wsite-content" class="wsite-elements wsite-not-footer">

<h1 class="wsite-content-title" style="text-align:left;"><font size="5">Bio</font></h1>

<div class="wsite-spacer" style="height:10px;"></div>
<div class="wsite-spacer" style="height:10px;"></div>

<div class="paragraph" style="text-align:left;">
	<span>
		I am a Ph.D. candidate in Electrical Engineering and Computer Science department at Massachusetts Institute of Technology. I am interested in developing machine learning models for functional genomics and therapeutic design. My advisor is <a href="cgs.csail.mit.edu">David Gifford</a>.

		<br><br>

		Before coming to MIT, I got my bachelor’s degree at <a href="http://www.tsinghua.edu.cn/publish/newthuen/index.html">Tsinghua University</a> in Beijing, China. I have worked with Professor <a href="https://profiles.stanford.edu/david-dill">David Dill</a> in Stanford in the summer of 2012. I have also been to <a href="http://www.utoronto.ca/">University of Toronto</a> as an exchange student.

	</span>
</div>

<div class="wsite-spacer" style="height:50px;"></div>


<!--- Research -->

<h1 class="wsite-content-title" style="text-align:left;"><font size="5">Research</font></h1>

<div class="wsite-spacer" style="height:10px;"></div>
<div class="wsite-spacer" style="height:10px;"></div>
<div class="paragraph" style="text-align:left;">


<h2 class="wsite-content-title" style="text-align:left;"><font size="4">Predict the functional impact of non-coding genetic variants</font></h2>
<span> 
	I developed machine learning models that accurately predict transcription factor binding and DNA methylation, two fundamental epigenetic phenotypes closely tied to gene regulation, from DNA sequence alone. The computationally predicted change in phenotype between the reference and alternate allele of a genetic variant accurately reflect its functional impact and improves the identification of regulatory variants causal for complex diseases.
</span>
<br><br>
   

<h2 class="wsite-content-title" style="text-align:left;"><font size="4">Model peptide display by the major histocompatibility complex</font></h2>
<span>
	I devised machine learning models that improve the prediction of peptides displayed by the major histocompatibility complex (MHC) on the cell surface. Computational modeling of peptide-display by MHC is central in the design of peptide-based therapeutics. Our machine learning models reduce false positives in high-affinity peptide design and improve the predictive accuracy in natural MHC ligands prediction.
</span>
<br><br>

    
<h2 class="wsite-content-title" style="text-align:left;"><font size="4">Design novel antibody sequences with improved affinity and specificty</font></h2>
<span>
	I developed machine learning frameworks to model the enrichment of an antibody sequence in phage-panning experiments against a target antigen. We show that antibodies with low specificity can be reduced by a computational procedure using machine learning models trained for multiple targets. Moreover, machine learning can help to design novel antibody sequences with improved affinity.
</span>
</div>





<!--- Publications -->

<div class="wsite-spacer" style="height:50px;"></div>

<h1 id="pub" class="wsite-content-title" style="text-align:left;"><font size="5">Publications</font> <a href="https://scholar.google.com/citations?user=5z2rh_oAAAAJ&hl=en"><font size="4">(Google Scholar)</font></a></h1>

<div class="paragraph" style="text-align:left;"> (* indicate co-first authors):<br><br>


    <a href=""><strong>
    	Accurate prediction of MHC class I ligands using peptide embedding
    </strong></a>
    <br>
    ​<strong>Zeng H</strong> and Gifford DK
    <br>
    <i>
   	<strong>Cell Systems</strong>
    </i>
     (to appear)
    <br><br>



    <a href=""><strong>
    	Peptide-MHC binding uncertainty estimation for therapeutic design
    </strong></a>
    <br>
    ​<strong>Zeng H</strong> and Gifford DK
    <br>
    <i>
   	Intelligent Systems for Molecular Biology (<strong>ISMB/ECCB</strong>)
    </i>
    , 2019
    <br>
    <i>
    <strong>Bioinformatics</strong>
    </i>
     (to appear)
    <br><br>



    <a href="https://arxiv.org/abs/1711.00141"><strong>
    	Training GANs with Optimism
    </strong></a>
    <br>
    ​Constantinos D*, Ilyas A*, Syrgkanis V*, and <strong>Zeng H</strong>*
    <br>
    <i>
   	International Conference on Learning Representations <strong>(ICLR)</strong>
    </i>
    , 2018
    <br><br>

    <a href="https://genome.cshlp.org/content/28/6/891.short"><strong>
    	A novel k-mer set memory (KSM) motif representation improves regulatory variant prediction
    </strong></a>
    <br>
    ​Guo Y, Tian K, <strong>Zeng H</strong>, Guo X, and Gifford DK.
    <br>
    <i>
   	<strong>Genome research</strong>
    </i>
    , 28 (6), 891-900, 2018
    <br><br>



    <a href="https://academic.oup.com/nar/article/3072752/Predicting"><strong>
    	Predicting the impact of non-coding variants on DNA methylation. 
    </strong></a>
    <br>
    ​<strong>Zeng H</strong>, and Gifford DK.
    <br>
    <i>
   	<strong>Nucleic Acids Research</strong>
    </i>
    , 45 (11): e99, 2017
    <br><br>



    <a href="http://onlinelibrary.wiley.com/doi/10.1002/humu.23198/abstract"><strong>
    	Accurate eQTL prioritization with an ensemble-based framework
    </strong></a>
    <br>
    ​<strong>Zeng H</strong>, Edwards MD, Guo Y, and Gifford DK. 
    <br>
    <i>
   	<strong>Human Mutation</strong>
    </i>
    , 38(9), 1259-1265, 2017
    <br><br>



    <a href="https://link.springer.com/book/10.1007/978-3-319-56970-3#page=387"><strong>
    	K-mer Set Memory (KSM) Motif Representation Enables Accurate Prediction of the Impact of Regulatory Variants. 
    </strong></a>
    <br>
    ​Guo Y, Tian K, <strong>Zeng H</strong>, and Gifford DK.
    <br>
    <i>
   	Research in Computational Molecular Biology (<strong>RECOMB</strong>)
    </i>
    , p. 372. Springer, 2017
    <br><br>



    <a href="http://onlinelibrary.wiley.com/doi/10.1002/humu.23197/abstract"><strong>
    	Predicting gene expression in massively parallele reporter assays: a comparative study 
    </strong></a>
    <br>
    ​Kreimer A, <strong>Zeng H</strong>, Edwards MD, Guo Y, Tian K, Shin S, Welch R, Wainberg M, Mohan R, Sinnott-Armstrong NA, Li Y, Eraslan G, AMIN TB, Goke J, Mueller NS, Kellis M, Kundaje A, Beer MA, Keles S, Gifford DK and Yosef N
    <br>
    <i>
   	<strong>Human Mutation</strong>
    </i>
    , 38(9), 1240-1250, 2017
    <br><br>



    <a href="http://bioinformatics.oxfordjournals.org/content/32/12/i121.short"><strong>
    	Convolutional Neural Network Architectures for Predicting DNA-Protein Binding. 
    </strong></a>
    <br>
    ​<strong>Zeng H</strong>, Edwards M, Liu G, and Gifford DK. 
    <br>
    <i>
    Intelligent Systems for Molecular Biology (<strong>ISMB</strong>)
    </i>,
    2016
    <br>
    <i>
    <strong>Bioinformatics</strong>
    </i>
    , 32(12), i121-i127, 2016
    <br><br>



    <a href="http://genome.cshlp.org/content/early/2016/07/25/gr.199778.115.abstract"><strong>
    	A DNA code governs chromatin accessibility
    </strong></a>
    <br>
    ​Hashimoto T, Sherwood RI, Kang DD, Barkal AA, <strong>Zeng H</strong>, Emons BJM, Srinivasan S, Rajagopal N, Jaakkola T, and Gifford DK. 
    <br>
    <i>
   	<strong>Genome Research</strong>
    </i>
    , 26(10), 1430-1440, 2016
    <br><br>



    <a href="http://bioinformatics.oxfordjournals.org/content/early/2015/11/05/bioinformatics.btv565"><strong>
    	GERV: A statistical method for generative evaluation of regulatory variants for transcription factor binding 
    </strong></a>
    <br>
    ​<strong>Zeng H</strong>, Hashimoto TB, Kang DD, and Gifford DK. 
    <br>
    <i>
   	<strong>Bioinformatics</strong>
    </i>
    , 32(4), 490-496, 2015
    <br><br>



    <a href="http://www.nature.com/ng/journal/vaop/ncurrent/full/ng.3461.html"><strong>
    	Abundant contribution of short tandem repeats to gene expression variation in humans 
    </strong></a>
    <br>
    ​Gymrek M, Willems T, Guilmatre A, <strong>Zeng H</strong>, Markus B, Georgiev S, Daly MJ, Price AL, Pritchard J, Sharp A and Erlich Y. 
    <br>
    <i>
   	<strong>Nature Genetics</strong>
    </i>
    , 48(1), 22–29, 2015
    <br><br>



    <a href="http://www.nature.com/nbt/journal/v32/n12/full/nbt.3052.html"><strong>
    	A community computational challenge to predict the activity of pairs of compounds 
    </strong></a>
    <br>
    ​Bansal M, Yang J, Karan C, Menden MP, Costello JC, Tang H et al. 
    <br>
    <i>
   	<strong>Nature Biotechnology</strong>
    </i>
    , 32(12), 1213-1222, 2014
    <br><br>



    <a href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0102119"><strong>
    	Mining TCGA data using Boolean implications 
    </strong></a>
    <br>
    ​Sinha S, Tsang EK, <strong>Zeng H</strong>, Meister M, and Dill DL. 
    <br>
    <i>
   	<strong>PLOS ONE</strong>
    </i>
    , 9(7): e102119, 2014
    <br><br>



    <a href="http://www.mit.edu/~haoyangz/files/ugvrPoster.pdf"><strong>
    	Integrative analysis of cancer data using Boolean implication networks.
    </strong></a>
    <br>
    ​<strong>Zeng H</strong>, Meister M, Sinha S, and Dill DL. 
    <br>
    <i>
   	<strong>RECOMB SB/RG/DREAM</strong>
    </i>
    , 2012
    <br><br>


<!--- Patent -->

<br>

<h1 id="pub" class="wsite-content-title" style="text-align:left;"><font size="5">Patents</font></h1>


    <a href="https://patents.google.com/patent/US20190065677A1/en"><strong>
    	Machine learning based antibody design
    </strong></a>
    <br>
    ​Gifford DK, <strong>Zeng H</strong>, Liu G
    <br>
    US Patent App. 16/171,596
    <br>


	
